### PR TITLE
Added two classes Marks and User. In main() there are no changes

### PR DESCRIPTION
**Pull request by VitalyCaraivanov**
/* Comment on pull request */

### CODE REVIEW

File: src/main/java/io/darkwings/User.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import java.util.Objects;
+
+public class User {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private int age;
+
+    Marks marks;
+    public User(int id, String firstName, String lastName, int age) {
```
Comment (michaels1999) [inline src/main/java/io/darkwings/User.java:12]: first of all , constructor without parameters needs to going firstly , after that you can create constructor with parameters ))

File: src/main/java/io/darkwings/User.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.age = age;
```
Comment (michaels1999) [inline src/main/java/io/darkwings/User.java:16]: where is prezent in constructor field for second object that you can create ? for marks?

**Review by michaels1999 — CHANGES_REQUESTED** [review]

File: src/main/java/io/darkwings/User.java (hunk 2 above)
Comment (VitalyCaraivanov) [reply src/main/java/io/darkwings/User.java:16]: My bad(

Comment (michaels1999) [reply src/main/java/io/darkwings/User.java:16]: now you need to amend commit and force push

File: src/main/java/io/darkwings/User.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+import java.util.Objects;
+
+public class User {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private int age;
+
+    Marks marks;
```
Comment (michaels1999) [inline src/main/java/io/darkwings/User.java:11]: provide setter and getters for marks also the another methods such as equals and hashcode , toString

**Pull request closed without merging**